subscription/models.py

```diff
@@ -25,6 +25,14 @@ class UserBucketUsage(models.Model):
     class Meta:
         db_table = 'UserBucketUsage'
 
+class PremiumUsageUnits(models.Model):
+    id = models.AutoField(primary_key=True)
+    url = models.CharField(max_length=255)
```

Comment:
Instead of adding URL here which can create a potential gap between the URL here vs the URL in the UriPattern table, we should add foreignKey to the UriPattern instead.